code-of-conduct/about-code-of-conduct-remediation-record.md

```diff
@@ -0,0 +1,99 @@
+* A Code of Conduct response plan template that you can use to create and explain the policy your team will follow as you handle Code of Conduct incidents. This template is included in the same folder as the Code of Conduct template and is named `template-code-of-conduct-response-plan.md`. See its accompany guide: `about-code-of-conduct-response-plan.md`.
+* A Code of Conduct incident record template for a form that is filled out when a community moderator takes an incident report from a community member. This template is included in the same folder as the Code of Conduct template as the Code of Conduct template and is named `template-code-of-conduct-incident-record.md`. See its accompany guide: `about-code-of-conduct-incident-record.md`.
+* A Code of Conduct remediation record template for a form that is filled out when a community moderator meets with a community member to explain the consequences of a Code of Conduct violation. This template is included in the same folder as the Code of Conduct template and is named `template-code-of-conduct-remediation-record.md`. See its accompany guide: `about-code-of-conduct-remediation-record.md`.
+
+You might also consider using the `../our-team/template-our-team.md` template to let your community members know who they can contact to report a Code of Conduct violation. This document is useful beyond Code of Conduct violations. It is a core document that helps you clearly communicate who belongs to your open source project or organization.
+---
+
+
+## Why do I need a Code of Conduct remediation record?
+
+See the `guide-code-of-conduct.md` for reasons to incorporate a Code of Conduct and response plan.
+
+The Code of Conduct remediation record is part of the documentation that needs to be kept when investigating and resolving Code of Conduct incidents.
+It is important to file this documentation to enable the community moderators to identify and prevent potential repeated patterns of abuse in the community.
+
+
+## Content of the Code of Conduct remediation record template
+
+The following sections provide guidance about how to fill out each section of the Code of Conduct remediation record template.
+
+:information_source: The Code of Conduct remediation record template includes boilerplate text that you can customize or adapt, use as-is, or completely replace with your own text.
+
+
+### About the "Introduction" section
+
+The *Introduction* section includes a description of the goals the community moderator should keep in mind while taking an incident report from an individual.
+The community moderator should refer to this section regularly to ensure that they stay focused on the goal of explaining the outcome of the Code of Conduct violation.
+These meetings have the potential to be stressful and full of high emotions.
+The introduction section provides some guidelines from keeping these meetings under control.

```

Comment:
suggestion (wording): Change to "...provides some guidelines _for_ keeping these meetings under control"